src/AssetsSelector.tsx

```diff
@@ -139,13 +139,8 @@ const AssetsSelector = ({
     )
 
     const getCameraPermissions = useCallback(async () => {
-        const { status: CAMERA }: any = await Permissions.askAsync(
-            Permissions.CAMERA
-        )
-
-        const { status: CAMERA_ROLL }: any = await Permissions.askAsync(
-            Permissions.MEDIA_LIBRARY
-        )
+        const { status: CAMERA }: any = await Permissions.askAsync("camera");
```

Comment:
are you sure about "cameraRoll" im looking on the expo library examples and i see that they prefer to use Alias like
Permission.CAMERA_ROLL etc 

im checking why its not working atm , also i see that you use "cameraRoll" while inside the types of media library i cant find it , can you explain how you get to this "cameraRoll" please ?

Reply:
in the Permissions.ts from mediaLibrary Api they said that "cameraRoll" is deprecated .

https://tinypng.com/web/output/w65agm7gd77h900nwx5j5kvxv53c03uv/roll.JPG

Reply:
I know it's weird, but my library still wasn't loading with the old permissions and when I tried changing it to cameraRoll, it suddenly worked. Android and iOS.

expo ^39.0.0
expo-permissions ^11.0.0

Reply:
ohh thats because your using Expo 39 , apparently expo team doing really good job by breaking older versions

the plugin should not work with Expo 39 im sorry , its working with the most updated version of Expo only.

This is something i cant control because they are changing stuff that force me to update too...

i just run iphone 8 with IOS 12+ and plugin works fine , it can display photos videos and compress images 

also expo-permissions 11 is yet not official that means also when u upgrade to expo 40 it will install expo permissions 10 and not 11


Please try to upgrade to expo 40 and update me if it wont work maybe we can find workaround 

maybe we let user send in the plugin the "permissions" so the plugin will ask base on what user sends.

Reply:
did u update to expo 40 and its resolve ?

Reply:
Alright, I will upgrade and try it out when I get the chance. Thanks for the info :)

Reply:
can u update if that helps ?